app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidator.java

```diff
@@ -0,0 +1,124 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class CardValidator {
+    public CardValidationInfo checkCardNumber(String input) {
```

Comment:
As I said, method should not take more than 10 lines. Only exceptions like "checking control sum" can be longer.